pyface/tests/test_image_resource.py

```diff
@@ -28,9 +30,9 @@
 )
 
 
-SEARCH_PATH = pkg_resources.resource_filename("pyface", "images")
-IMAGE_DIR = pkg_resources.resource_filename(__name__, "images")
-IMAGE_PATH = os.path.join(IMAGE_DIR, "core.png")
+SEARCH_PATH = str(files("pyface") / "images")
```

Comment:
I think the "right" way to convert a `Path`-like to a string is `os.fspath`.
```suggestion
SEARCH_PATH = os.fspath(files("pyface") / "images")
```

That's assuming that we actually want a string and that a `Path` won't work better.

Reply:
> That's assuming that we actually want a string and that a Path won't work better.

The rest of the package isnt happy with using `Path`s which is why I converted them to filesystem paths.